deployment/utils/ipAddress/configureStaticIpAddress.inc

```diff
@@ -0,0 +1,62 @@
+#
+# ******************************************************************************
+# Copyright (c) 2019 Bosch Software Innovations GmbH.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# https://www.eclipse.org/org/documents/epl-2.0/index.php
+#
+# *****************************************************************************
+#
+
+# Reads a static IP address from an environment variable following a convention
+# and updates a service configuration file.
+#
+# Note that it is assumed that the YAML file describes a single service only;
+# the service name is not used to select a service from the file.
+#
+# Arguments
+# 1. Path to Service configuration YAML file
+# 2. Service name
+function configureStaticIpAddress() {
```

Comment:
Hello, 

Is there a special reason for using the 'function' keyword?
It's optional and not POSIX compliant, and even if I can read from shebangs that you target bash, it's the only problem that I have running those scripts with zsh. 

Thanks a lot.